app/src/main/res/values-sv/strings.xml

```diff
@@ -90,7 +90,19 @@
     <string name="menu_item_title_share_session">Dela händelse</string>
     <string name="menu_item_title_share_favorites">Deka favoriter</string>
     <string name="schedule_changes_dialog_changed_new_cancelled_text">. Det är ändringar i <xliff:g example="3 lectures" id="num_changes">%1$s</xliff:g>, <xliff:g example="2 are" id="num_new">%2$s</xliff:g> new, <xliff:g example="1 is" id="num_canceled">%3$s</xliff:g> cancelled.</string>
-    <string name="schedule_changes_dialog_affected_text">Dessa ändringar påverkar <xliff:g example="2" id="num_marked">%1$d</xliff:g> av dina favoriserade lektioner.</string>
+    <string name="schedule_changes_dialog_affected_text">Dessa ändringar påverkar <xliff:g example="2" id="num_marked">%1$d</xliff:g> av dina favoriserade föreläsningar.</string>
+    <plurals name="schedule_changes_dialog_number_of_sessions">
+        <item quantity="one">%d föreläsning</item>
+        <item quantity="few">%d föreläsningar</item>
+        <item quantity="many">%d föreläsningar</item>
+        <item quantity="other">%d föreläsningar</item>
+    </plurals>
+    <plurals name="schedule_changes_dialog_being">
+        <item quantity="one">%d</item>
+        <item quantity="few">%d</item>
+        <item quantity="many">%d</item>
+        <item quantity="other">%d</item>
```

Comment:
OH. Is there no verb for "being" in Swedish?

Reply:
Yes, fixed.